pkg/services/folder/model.go

```diff
@@ -21,6 +21,7 @@ const (
 	GeneralFolderUID     = "general"
 	RootFolderUID        = ""
 	MaxNestedFolderDepth = 4
+	SharedFolderUID      = "sharedwithme"
```

Comment:
We need to think about UID for this folder. It should not be easy to hit it accidentally when you create new folder.

Reply:
I think we need to forbid creating a folder with this UID too

Reply:
folders are uniquely identified by `org_id` and `uid` so if we associate it with a non existing `org_id` (eg zero) then it will be probably fine